scene/gui/line_edit.cpp

```diff
@@ -926,7 +926,9 @@ void LineEdit::_notification(int p_what) {
 		} break;
 		case NOTIFICATION_FOCUS_ENTER: {
 			if (caret_blink_enabled) {
```

Comment:
Need to wrap this in `if (!caret_force_displayed) { if (caret_blink_enabled) { ... } else { ... } }`